lib/mssql.js

```diff
@@ -499,8 +499,8 @@ MsSQL.prototype.toDatabase = function (prop, val, wrap) {
         && typeof val.length === 'number') { //if value is array
         //check if it is an array of string, because in that cause we need to
         // wrap them in single quotes
-        if (typeof val[0] === 'string') {
```

Comment:
Should we do some sort of type checking? Removing this seems a bit dangerous at first glance.